### ISSUE TITLE
Trouble Selling Items

### ISSUE THREAD

**Issue by priddyma** [Maxx53/SCMBot#585]
int input = Convert.ToInt32(strInput); line returns a FormatException was unhandled. Occurs when trying to place the item for sale with a "Buyer Pays" of $0.20. 
The value of strInput at time of failure is: "$020".

**Comment by Maxx53** [comment]
yep, price parsing bug. just remove "$" char from price input textbox before pressing "set price" button

**Comment by priddyma** [comment]
Doesn't seem to work and gives no exception. Log file also does not show anything relating to the selling or being successful or failing.
